src/components/Authclient/CAuthclientEditorInfo.vue

```diff
@@ -431,11 +445,16 @@ export default {
   },
 
   props: {
-    authclient: {
+    resource: {
       type: Object,
       required: true,
     },
 
+    canDelete: {
+      type: Boolean,
+      default: () => false,
```

Comment:
You don't need to use an arrow function for types that are not passed as a reference (objects, arrays, maps, ...)